src/lizaalert/users/models.py

```diff
@@ -321,3 +334,25 @@ def create_default_volunteer_level(sender, instance, created, **kwargs):
 
             if not existing_record:
                 VolunteerLevel.objects.create(volunteer=volunteer, level=beginner_level, confirmed=True)
+
+
+class UserDivisionLevel(models.Model):
+    """Модель для связи пользователей, направлений и уровней."""
+
+    volunteer = models.ForeignKey(
+        Volunteer,
+        on_delete=models.PROTECT,
+        related_name="divisionlevels",
+        verbose_name="Волонтер",
+    )
+    division = models.ForeignKey("courses.Division", on_delete=models.PROTECT, verbose_name="Направление умения")
+    level = models.ForeignKey(
+        Level, on_delete=models.PROTECT, verbose_name="Уровень умения", default=Level.LevelName.beginner
+    )
```

Comment:
из этой таблицы можно удалить записи

```suggestion
    division = models.ForeignKey("courses.Division", on_delete=models.CASCADE_DELETE, verbose_name="Направление умения")
    level = models.ForeignKey(
        Level, on_delete=models.CASCADE_DELETE, verbose_name="Уровень умения", default=Level.LevelName.beginner
    )
```

Reply:
Установил CASCADE